src/main/java/net/minecraftforge/fmp/capabilities/CapabilityWrapperRegistry.java

```diff
@@ -0,0 +1,43 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.Collection;
+import java.util.IdentityHashMap;
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.items.CapabilityItemHandler;
+
+public class CapabilityWrapperRegistry
+{
+    
+    private static Map<String, ICapabilityWrapper<?>> wrappers = new IdentityHashMap<String, ICapabilityWrapper<?>>();
+
```

Comment:
This needs an explination, how and where is it used? Why would people wrap the factories? Better to call them actualk factories, but whatever